crates/apps/src/lib/node/ledger/shell/process_proposal.rs

```diff
@@ -693,6 +697,69 @@ mod test_process_proposal {
 
     const GAS_LIMIT_MULTIPLIER: u64 = 100_000;
 
+    /// Check that we reject a validator set update protocol tx
+    /// if the bridge is not active.
+    #[test]
+    fn check_rejected_valset_upd_bridge_inactive() {
```

Comment:
should we also check that votes on bridge pool roots and eth events are rejected?

Reply:
we do already, in `check_rejected_bp_roots_bridge_inactive` and `check_rejected_eth_events_bridge_inactive`